man1/aur.1

```diff
@@ -380,22 +380,20 @@ in $PATH, for example
 .BR aur\-remove
 
 .EX
-  #!/bin/bash
-  # alternative to 'repoctl remove' which loops over all local repositories
-  package=$1
+  #!/bin/sh --
+  # aur-remove - remove listed packages from all local repositories
 
-  aur repo --repo-list | while read -r repo_path; do
-    # remove package from database
-    repo-remove "$(readlink -e "$repo_path")" "$package"
+  if [ "$#" -eq 0 ]; then
+    printf 'usage: aur remove package [package ...]\en' >&2
+    exit 1
+  fi
 
-    # remove all versions of package
-    find "$(dirname "$repo_path")" -name "$package*" -delete
+  aur repo --repo-list | while read -r repo_path; do
+    if repo_path=$(readlink -e "$repo_path"); then
+      repo-remove "$repo_path" "$@"
+      paccache -c "${repo_path%/*}" -rvk0 "$@"
+    fi
   done
-
-  # remove package from system
```

Comment:
I guess this is left to the user, or are the packages supposed to linger as foreign packages?

Reply:
It is a bit out of scope for a repo-management script to remove the packages on the host. 

This could be addressed by implementing `--uninstal` for example, but in the current context this is inserted, which is to show a minimal example of the custom script feature, that is left to the user as any other change they see fit.

Also missing from the script, and maybe also out of scope, is the removal of files in`.cache/aurutils/sync`.

cc: @Earnestly

Reply:
Stop trying to conflate repo management and package management